showcase/compliance_test.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2021 Google LLC
```

Comment:
Do you know why this file is showing up as "new"? It was introduced in https://github.com/googleapis/gapic-generator-go/pull/625....

Reply:
This PR is against `master`

Reply:
What? No, that's totally wrong.

Reply:
Changed the base to diregapic-dev-branch. I think that clears this up.